package.json

```diff
@@ -1,10 +1,13 @@
 {
   "name": "amcharts3-react",
-  "version": "1.1.8",
+  "version": "1.1.9",
```

Comment:
I prefer to put multiple changes into a single update, so I prefer to bump the version number when releasing, not when making the changes.